frontend/src/components/match/stats/EditMinStats.js

```diff
@@ -66,27 +66,39 @@ const EditStats = () => {
     mutationFn: matchStatsHalfTime,
     onSuccess: () => {
       queryClient.invalidateQueries({ queryKey: ["match", params.matchId] });
+      queryClient.invalidateQueries({
+        queryKey: ["match-stats", params.matchId]
+      });
```

Comment:
possible to pull this out into a function, same thing we are using 5-6 times through the page